tests/vibe.http.server.listenHTTP/source/app.d

```diff
@@ -0,0 +1,22 @@
+import vibe.core.core;
+import vibe.core.core;
+import vibe.core.log;
+import vibe.http.client;
+import vibe.http.server;
+import vibe.stream.operations : readAllUTF8;
+
+shared static this()
+{
+	listenHTTP(":11721", (scope req, scope res) {
```

Comment:
I copied the hard-coded port from other tests - how about adding something like [`getFreePort`](https://github.com/dlang-bots/dlang-bot/blob/master/test/utils.d#L26) to Vibe.d?